internal/utils/endorsement.go

```diff
@@ -0,0 +1,42 @@
+package utils
+
+import (
+	"fmt"
+
+	"github.com/hyperledger/fabric-protos-go/peer/lifecycle"
+	"github.com/hyperledger/fabric/bccsp/factory"
+	"github.com/hyperledger/fabric/common/cauthdsl"
+	"github.com/hyperledger/fabric/msp"
+	"github.com/hyperledger/fabric/msp/mgmt"
+)
+
+func NewPolicyEvaluator() *PolicyEvaluator {
```

Comment:
Do we need PolicyEvaluator in its function public given that we have IsValidEndorserIdentity?